external-resolver/src/main/scala/org/genivi/sota/resolver/Boot.scala

```diff
@@ -78,21 +84,39 @@ class Routing(db: Database)
       } ~
       (post & entity(as[Filter])) { filter => complete(db.run(Filters.add(filter)))
       } ~
-      (put  & refined[Filter.ValidName](PathMatchers.Slash ~ PathMatchers.Segment ~ PathMatchers.PathEnd)
+      (put  & refined[Filter.ValidName](Slash ~ Segment ~ PathEnd)
             & entity(as[Filter.Expression])) { (fname, expr) =>
         handleExceptions(packageFiltersHandler) {
           complete(db.run(Filters.update(Filter(fname, expr))))
         }
+      } ~
+      (delete & refined[Filter.ValidName](Slash ~ Segment ~ PathEnd)) { fname =>
+        handleExceptions(packageFiltersHandler) {
+          complete(db.run(Filters.delete(fname)))
+        }
       }
     }
 
-  def validateRoute: Route =
+  def validateRoute: Route = {
     pathPrefix("validate") {
       path("filter") ((post & entity(as[Filter])) (_ => complete("OK")))
     }
+  }
 
   def packageFiltersRoute: Route = {
 
+    def packageFiltersHandler: ExceptionHandler = ExceptionHandler {
```

Comment:
This is actually old code which I just moved, but I think during rebase I accidentally reintroduced the old code.